### PR TITLE
Textarea & date

### PR DESCRIPTION
**Pull request by danicadombek**
Create textarea and datepicker in CameraPage and add the dates to DetailMemoryPage.

### CODE REVIEW

File: src/GlobalStyles.js [hunk 1]
```diff
@@ -4,6 +4,8 @@ import { createGlobalStyle } from 'styled-components'
 export default createGlobalStyle`
 :root {
   --border-radius-global: 50px;
+  --border-radius-form: 20px;
+  --border-radius-form: 10px;
```
Comment (macfrei) [inline src/GlobalStyles.js]: The second value `--border-radius-form` is overwriting the first one. You only need this once!

File: src/pages/CameraPage.js [hunk 2]
```diff
@@ -38,13 +38,24 @@ export default function CameraPage({ onNavigate, handleMemorySubmit }) {
...
+        <TitleDate>
+          <InputTitle
+            aria-label="Choose a name"
+            id="upload-img"
+            type="text"
+            name="title"
+            placeholder="Choose a name"
+            autoComplete="off"
+            maxlength="30"
+          />
```
Comment (macfrei) [inline src/pages/CameraPage.js]: Where is your `label` element?

**Review by macfrei — COMMENTED** [review]
Nice job! You should probably update the stories and tests as well, right? 🚀

File: src/pages/CameraPage.js (hunk 2 above)
Comment (danicadombek) [reply src/pages/CameraPage.js]: Jerry told me I can use the aria-label instead.

File: src/GlobalStyles.js (hunk 1 above)
Comment (danicadombek) [reply src/GlobalStyles.js]: Oh yes :)

**Pull request merged**